internal/database/exposure.go

```diff
@@ -145,12 +145,12 @@ func generateExposureQuery(criteria IterateExposuresCriteria) (string, []interfa
 
 	if !criteria.SinceTimestamp.IsZero() {
 		args = append(args, criteria.SinceTimestamp)
-		q += fmt.Sprintf(" AND created_at > $%d", len(args))
+		q += fmt.Sprintf(" AND created_at >= $%d", len(args))
```

Comment:
Add a comment about why >= and <

this was a subtle bug you found.

Reply:
done.